File: WebSolution/PMSNGApp/src/app/task/Models/task.ts
export class task{
    constructor(public id:number,
        public title:string,
        public description:string,
        public assignDate:string,
        public startDate:string,
        public dueDate:string,
        public assignedTo:number,
        public assignedBy:number,
        public projectId:number,
        public status:string){

    }
}

File: WebSolution/Monolithic/Intranet/Models/Task.cs
namespace TFLPortal.Models;
public class Task
{
   public int TaskId{get;set;}
   public string? Title{get;set;}
   public string? TaskType{get;set;}
   public string? Description{get;set;}
   public DateTime AssignDate{get;set;}
   public DateTime StartDate{get;set;}
   public DateTime DueDate{get;set;}
   public DateTime CreatedDate{get;set;}
   public string? Status{get;set;}

   public int AssignedTo{get;set;}  //AssignedTo
   public int AssignedBy{get;set;}  //AssignedBy
   public int ProjectId{get;set;}  //ProjectId
   public int SprintId{get;set;}//SprintId    
}
